.github/bump_chart_versions.sh

```diff
@@ -51,6 +51,6 @@ if [[ "$CHART" == "redpanda" ]]; then
 fi
 
 bump_patch_version
-sed -i "s@^version: .*\$@version: ${NEW_VERSION}@" "${CHARTFILE}"
-sed -i "s@^appVersion: .*\$@appVersion: ${LATEST}@" "${CHARTFILE}"
-sed -i "s@${REPO}:${APPVERSION}\$@${REPO}:${LATEST}@" "${CHARTFILE}"
+perl -i -pe "s@^version: .*\$@version: ${NEW_VERSION}@" "${CHARTFILE}"
+perl -i -pe "s@^appVersion: .*\$@appVersion: ${LATEST}@" "${CHARTFILE}"
+perl -i -pe "s@${REPO}:${APPVERSION}\$@${REPO}:${LATEST}@" "${CHARTFILE}"
```

Comment:
question: why are we going replace perl for sed? I like perl, but find it not always being installed on the modern OSs by default. sed is usually always available.

Reply:
In the MacOS sed with in place replacement doesn't work correctly. I'm working on Mac, so I needed this script to work.

Based on https://stackoverflow.com/questions/4247068/sed-command-with-i-option-failing-on-mac-but-works-on-linux I tested `sed -i'' -e ...`, but it created backup file which is useless (named `Chart.yaml-e`). That's why I switched to perl.

Reply:
`brew install gnu-sed` and create an alias to it or put /usr/local/bin first in your PATH.

Reply:
From what I see,  using `sed -i '.bak' -e ...` might work to not create that cryptic file extension?, THough I see, that if this additionally created backup file is causing issues during the build and the `perl` approach works, let's go with perl :-)  I have approved. Thank you!